fix/fix.cpp

```diff
@@ -100,9 +97,6 @@ void InitMathTables() {
 
   asin_table[256] = asin_table[255];
   acos_table[256] = acos_table[255];
-
-  //	Initialize a random seed.
-  ps_srand(time(NULL));
```

Comment:
So this is not needed to initialize the random number generator?

Reply:
they relocated seed initialization to init.cpp

Reply:
Random seed was injected into InitMathTables when it was part of vecmat submodule. Logically seed initialization not belongs to vecmat or fixmat components so I decided to move it out to common initialization function.